pkg/cli/environment.go

```diff
@@ -30,13 +30,17 @@ import (
 
 	"github.com/spf13/pflag"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/rest"
 
 	"helm.sh/helm/v3/pkg/helmpath"
 )
 
 // defaultMaxHistory sets the maximum number of releases to 0: unlimited
 const defaultMaxHistory = 10
 
+// defaultBurstLimit sets the default client-side throttling limit
+const defaultBurstLimit = 250
```

Comment:
why is 250?

Reply:
This is the same value OpenShift's `oc` is using as default, see https://github.com/helm/helm/issues/10560.

Reply:
ok